Sources/FaceLiveness/FaceDetection/BlazeFace/FaceDetectorShortRange+Model.swift

```diff
@@ -105,10 +111,17 @@ extension FaceDetectorShortRange {
                     count: confidenceScoresCapacity
                 )
             )
+            
+            let blazeFaceDetectionThreshold: Float
+            if let sessionConfiguration = faceDetectionSessionConfiguration?.getFaceDetectionSessionConfiguration() {
+                blazeFaceDetectionThreshold = Float(sessionConfiguration.ovalMatchChallenge.faceDetectionThreshold)
+            } else {
+                blazeFaceDetectionThreshold = confidenceScoreThreshold
+            }
 
             var passingConfidenceScoresIndices = confidenceScores
                 .enumerated()
-                .filter { $0.element >= confidenceScoreThreshold }
```

Comment:
is `confidenceScoreThreshold` a magic number that needs to be removed?

Reply:
yes, this value is BlazeFaceDetectionThreshold